tests/zero_code_change/pt_utils.py

```diff
@@ -5,10 +5,14 @@
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
+import torch.optim as optim
 import torchvision
 import torchvision.transforms as transforms
 from packaging import version
 
+# First Party
+import smdebug.pytorch as smd
```

Comment:
nit (non-blocking): maybe use a different variable name to make it clear that this is the PyTorch module of smdebug?

Reply:
Agreed but this is consistent with the way all of the imports of the framework module are done. Unless we take a decision to refactor all of the files in smdebug and the AWS framework codebases, it's better to just be consistent with the naming.